joomla/joomla-cms
# [4.0] Cleanup the View file com_cache

### Summary of Changes
This is the next step in cleaning up the view files and adhering to coding standards as well as making the views more usable by the IDE by removing the magic getter.

### Testing Instructions

1. Go to System -> Clear Cache
2. Make sure Cache page loads as before

### Documentation Changes Required
None
